### PR TITLE
[zh]Sync reference-11

### PR DESCRIPTION
**Pull request by nameYULI**
related issue: https://github.com/kubernetes/website/issues/33478
reference-11
```
30	19	content/en/docs/reference/node/topics-on-dockershim-and-cri-compatible-runtimes.md
```

### CODE REVIEW

File: content/zh/docs/reference/node/topics-on-dockershim-and-cri-compatible-runtimes.md [hunk 1]
```diff
@@ -1,83 +1,99 @@
...
+* Kubernetes enhancement proposal: [KEP-2221: Removing dockershim from kubelet](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/2221-remove-dockershim/README.md)
 
-* [Kubernetes Blog: "Dockershim removal is coming. Are you ready?", 2021/11/12](/blog/2021/11/12/are-you-ready-for-dockershim-removal/)
+* Kubernetes enhancement proposal issue: [Removing dockershim from kubelet](https://github.com/kubernetes/enhancements/issues/2221) (_k/enhancements#2221_)
 -->
 
-## 首要来源
+## Kubernetes 项目
+
+* Kubernetes 博客：[Dockershim 弃用常见问题解答](/blog/2022/02/17/dockershim-faq/)（最初发表于 2022/02/17）
```
Comment (Sea-n) [inline content/zh/docs/reference/node/topics-on-dockershim-and-cri-compatible-runtimes.md:44]: This FAQ has Mandarin version.
<https://kubernetes.io/zh/blog/2022/02/17/dockershim-faq/>

File: content/zh/docs/reference/node/topics-on-dockershim-and-cri-compatible-runtimes.md [hunk 2]
```diff
@@ -1,83 +1,99 @@
...
+
+* Kubernetes 博客：[Kubernetes 即将移除 Dockershim：承诺和下一步](/blog/2022/01/07/kubernetes-is-moving-on-from-dockershim/)（发表于 2022/01/07）
 
-* [Kubernetes 博客: “Dockershim 弃用常见问题解答”, 2020/12/02](/blog/2020/12/02/dockershim-faq/)
+* Kubernetes 博客：[移除 Dockershim 即将到来。你准备好了吗？](/blog/2021/11/12/are-you-ready-for-dockershim-removal/)（发表于  2021/11/12）
```
Comment (Sea-n) [inline content/zh/docs/reference/node/topics-on-dockershim-and-cri-compatible-runtimes.md:48]: This article has Mandarin version too.
<https://kubernetes.io/zh/blog/2022/01/07/kubernetes-is-moving-on-from-dockershim/>

File: content/zh/docs/reference/node/topics-on-dockershim-and-cri-compatible-runtimes.md [hunk 3]
```diff
@@ -1,83 +1,99 @@
...
 
-* [Kubernetes enhancement proposal: "KEP-2221: Removing dockershim from kubelet"](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/2221-remove-dockershim/README.md)
+* Kubernetes enhancement proposal: [KEP-2221: Removing dockershim from kubelet](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/2221-remove-dockershim/README.md)
 
-* [Kubernetes Blog: "Dockershim removal is coming. Are you ready?", 2021/11/12](/blog/2021/11/12/are-you-ready-for-dockershim-removal/)
+* Kubernetes enhancement proposal issue: [Removing dockershim from kubelet](https://github.com/kubernetes/enhancements/issues/2221) (_k/enhancements#2221_)
 -->
 
-## 首要来源
+## Kubernetes 项目
```
Comment (Sea-n) [inline content/zh/docs/reference/node/topics-on-dockershim-and-cri-compatible-runtimes.md:42]: The link anchor `{#kubernetes-project}` should be added.

**Review by Sea-n — COMMENTED** [review]
Here is some nitpick for the PR, please take a look. 🙃

File: content/zh/docs/reference/node/topics-on-dockershim-and-cri-compatible-runtimes.md [hunk 4]
```diff
@@ -1,83 +1,99 @@
...
+This is a list of articles and other pages that are either
+about the Kubernetes' deprecation and removal of _dockershim_,
+or about using CRI-compatible container runtimes, in connection
+with that removal.
 -->
-这是有关以下内容的文章列表：
-
-- Kubernetes 弃用和删除 _dockershim_
-- 使用兼容 CRI 的容器运行时
+这是关于 Kubernetes 弃用和删除 “dockershim”
```
Comment (tengqm) [inline content/zh/docs/reference/node/topics-on-dockershim-and-cri-compatible-runtimes.md:20]:
```suggestion
本页是关于 Kubernetes 弃用和删除 “dockershim”
```

**Review by tengqm — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**